docs/docs/segment-executiontime.md

```diff
@@ -0,0 +1,29 @@
+---
+id: executiontime
+title: Execution Time
+sidebar_label: Execution Time
+---
+
+## What
+
+Displays the execution time of the previously executed command.
```

Comment:
Interesting to add here is how to use it. Given you either need to use the PowerShell module (or set it up manually by using the snippet), it might be handy to know this is needed as a prerequisite. There's information on how to do this in [zsh](https://gist.github.com/knadh/123bca5cfdae8645db750bfb49cb44b0) and bash (see the accepted answer on [this post](https://stackoverflow.com/questions/2704635/is-there-a-way-to-find-the-running-time-of-the-last-executed-command-in-the-shel)) as well.

Reply:
* I have updated schema.json
* I have added schema.json information to the contribution docs
* I have updated the zsh instructions
  * I took the liberty to rename your functions and namespace everything with "omp". Let me know if you dislike that change.

Later today, I'll look into bash and also make the segment-execution.md changes. I also want to change this documentation to use a different icon, since the one I selected is one that is bugged on powerline segments.

Reply:
Looks good to me, love the addition of the `omp` renames, makes sense (I initially copied those from elsewhere).

Reply:
@JanDeDobbeleer I had trouble getting the execution time with bash. I did find some scripts that give you the zshell-style hooks, but they are quite long, and would likely complicate the bash installation quite a bit. [Here is an example](https://github.com/rcaloras/bash-preexec/blob/master/bash-preexec.sh).

I'm thinking we include a general description stating this segment works with the Powershell module, or with an execution-time parameter passed in. And provide the installation instructions for Powershell and Zshell. I have updated the docs in this manner. Please have a look and see what you think.